shared/django_apps/legacy_migrations/migrations/0006_delete_many_owner_triggers.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.16 on 2024-11-19 21:07
+
+from django.db import migrations
+
+from shared.django_apps.migration_utils import RiskyRunSQL
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("legacy_migrations", "0005_delete_branch_update_db_trigger"),
+    ]
+
+    operations = [
+        RiskyRunSQL(
+            """
+            DROP TRIGGER IF EXISTS owner_yaml_updated ON owners;
+            DROP TRIGGER IF EXISTS owner_cache_state_update ON owners;
+            DROP TRIGGER IF EXISTS owner_cache_state_insert ON owners;
+            DROP TRIGGER IF EXISTS owner_token_clered ON owners;
```

Comment:
lol, love the typo that has survived all this time :-D

Reply:
"Naming is hard" - someone in the past 😂